x-pack/plugin/ql/src/main/java/org/elasticsearch/xpack/ql/type/DataTypeConverter.java

```diff
@@ -334,15 +363,61 @@ public static int safeToInt(long x) {
         return (int) x;
     }
 
-    public static long safeToLong(double x) {
+    public static long safeDoubleToLong(double x) {
```

Comment:
The original method didn't get renamed, strictly "speaking", but changed to take a Number. But also needed to add a new one (safeDoubleToLong) that takes doubles only. Since they both take doubles, I wanted to avoid the ambiguity.